### DISCUSSION TITLE
If I made a fork, can I change name in MIT License?

### DISCUSSION THREAD

**Discussion by uuidev** [community/community#47161] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello!

I've forked one repository and now I'm going to create something other based on the previous code. Do I need to change the name in the LICENSE file where the copyright is specified?

If my question doesn't fit here, please direct me where it should.

Thank you!

**Comment by OwaisAzad** [comment, marked as answer]
Yes, if you're creating a new project based on code from a forked repository, you should update the copyright information in the LICENSE file to reflect the changes. The LICENSE file typically specifies the copyright holder for the code, and you should make sure that the information is accurate and up-to-date for your new project.

It's also a good idea to consult the specific license terms for the code in the LICENSE file to see if there are any specific requirements or restrictions that you need to follow. Some open-source licenses, such as the MIT License, allow for modification and reuse of the code as long as the original copyright and license information are preserved. Other licenses, such as the GPL, have more restrictive requirements and may require you to release your code under the same license.

**Reply by uuidev** [reply]
Thank you for your answer!

**Reply by queenofcorgis** [reply]
Yes - thank you @OwaisAzad for such a robust and helpful answer. Also @uuidev appreciate you marking this as solved to help any other folks who may be. wondering the same ✅
